silkrpc/http/request_handler.cpp

```diff
@@ -24,86 +24,104 @@
-    {method::k_eth_getLogs, &commands::RpcApi::handle_eth_get_logs},
-    {method::k_eth_sendRawTransaction, &commands::RpcApi::handle_eth_send_raw_transaction},
-    {method::k_eth_sendTransaction, &commands::RpcApi::handle_eth_send_transaction},
-    {method::k_eth_signTransaction, &commands::RpcApi::handle_eth_sign_transaction},
-    {method::k_eth_getProof, &commands::RpcApi::handle_eth_get_proof},
-    {method::k_eth_mining, &commands::RpcApi::handle_eth_mining},
-    {method::k_eth_coinbase, &commands::RpcApi::handle_eth_coinbase},
-    {method::k_eth_hashrate, &commands::RpcApi::handle_eth_hashrate},
-    {method::k_eth_submitHashrate, &commands::RpcApi::handle_eth_submit_hashrate},
-    {method::k_eth_getWork, &commands::RpcApi::handle_eth_get_work},
-    {method::k_eth_submitWork, &commands::RpcApi::handle_eth_submit_work},
-    {method::k_eth_subscribe, &commands::RpcApi::handle_eth_subscribe},
-    {method::k_eth_unsubscribe, &commands::RpcApi::handle_eth_unsubscribe},
-    {method::k_eth_getBlockReceipts, &commands::RpcApi::handle_parity_get_block_receipts},
-    {method::k_debug_accountRange, &commands::RpcApi::handle_debug_account_range},
-    {method::k_debug_getModifiedAccountsByNumber, &commands::RpcApi::handle_debug_get_modified_accounts_by_number},
-    {method::k_debug_getModifiedAccountsByHash, &commands::RpcApi::handle_debug_get_modified_accounts_by_hash},
-    {method::k_debug_storageRangeAt, &commands::RpcApi::handle_debug_storage_range_at},
-    {method::k_debug_traceTransaction, &commands::RpcApi::handle_debug_trace_transaction},
-    {method::k_debug_traceCall, &commands::RpcApi::handle_debug_trace_call},
-    {method::k_trace_call, &commands::RpcApi::handle_trace_call},
-    {method::k_trace_callMany, &commands::RpcApi::handle_trace_call_many},
-    {method::k_trace_rawTransaction, &commands::RpcApi::handle_trace_raw_transaction},
-    {method::k_trace_replayBlockTransactions, &commands::RpcApi::handle_trace_replay_block_transactions},
-    {method::k_trace_replayTransaction, &commands::RpcApi::handle_trace_replay_transaction},
-    {method::k_trace_block, &commands::RpcApi::handle_trace_block},
-    {method::k_trace_filter, &commands::RpcApi::handle_trace_filter},
-    {method::k_trace_get, &commands::RpcApi::handle_trace_get},
-    {method::k_trace_transaction, &commands::RpcApi::handle_trace_transaction},
-    {method::k_tg_getHeaderByHash, &commands::RpcApi::handle_tg_get_header_by_hash},
-    {method::k_tg_getHeaderByNumber, &commands::RpcApi::handle_tg_get_header_by_number},
-    {method::k_tg_getLogsByHash, &commands::RpcApi::handle_tg_get_logs_by_hash},
-    {method::k_tg_forks, &commands::RpcApi::handle_tg_forks},
-    {method::k_tg_issuance, &commands::RpcApi::handle_tg_issuance},
-    {method::k_parity_getBlockReceipts, &commands::RpcApi::handle_parity_get_block_receipts},
-};
+std::map<std::string, RequestHandler::HandleMethod> RequestHandler::handlers_;
+
+void RequestHandler::add_debug_handlers() {
```

Comment:
We probably need to make `handlers_` non-static here

Reply:
Yes, you're right, we will in another PR. This is just a refactoring in preparation for that